bigbluebutton-html5/imports/ui/components/app/styles.scss

```diff
@@ -9,6 +9,7 @@
   --bars-padding: calc(var(--lg-padding-x) - .45rem); // -.45 so user-list and chat title is aligned with the presentation title
   --userlist-handle-width: 5px; // 5px so user-list and chat resize handle render as the same size
   --poll-pane-min-width: 20em;
+  --chat-margin-right: 0.1em;
```

Comment:
Since the margin is being applied to the other panels as well, not only the chat.  changing the name to something more generic like `--panel-margin-right` might be a good idea.